src/core/dbus.c

```diff
@@ -1004,8 +1008,19 @@ static void destroy_bus(Manager *m, sd_bus **bus) {
         }
 
         /* Get rid of tracked clients on this bus */
-        if (m->subscribed && sd_bus_track_get_bus(m->subscribed) == *bus)
+        if (m->subscribed && sd_bus_track_get_bus(m->subscribed) == *bus) {
+                _cleanup_strv_free_ char **subscribed = NULL;
+                int r;
+
+                r = bus_track_to_strv(m->subscribed, &subscribed);
+                if (r < 0)
+                        log_error_errno(r, "Failed to serialize api subscribers: %m");
```

Comment:
i think it is fine to ignore failures here, but in that case downgrade this to `log_warning`, and say in the log message that you are ignoring this failure, i.e. end it in `… subscribers, ignoring: %m");`